src/main/java/stringcalculator/domain/Operator.java

```diff
@@ -0,0 +1,49 @@
+package stringcalculator.domain;
+
+import stringcalculator.common.ErrorMessage;
+
+import java.util.Arrays;
+import java.util.function.BiFunction;
+
+public enum Operator {
+    PLUS("+", (number, operand) -> Operator.add(number, operand)),
```

Comment:
[메서드 참조](https://jinseongsoft.tistory.com/208)를 통해 아래와 같이 작성할 수 있습니다 :) 

```suggestion
    PLUS("+", Operator::add),
```

Reply:
enum과 함수형 인터페이스를 공부할 때도 무척 재밌었지만, 이런식으로 코드 가독성을 높힐 수 있는 방법에 대해 알아가는 것도 정말 재밌네요. 

특히, 함수형 인터페이스에서 람다를 다루고 이를 apply로 값을 내보내 주었는데 오히려 람다가 많아지면 코드의 가독성이 떨어질 수 있다는 생각을 못했습니다. 

감사합니다. :)